routes/api.php

```diff
@@ -1,8 +1,8 @@
 <?php
 
-// use Illuminate\Http\Request;
-// use Illuminate\Support\Facades\Route;
-//
-// Route::middleware('auth:api')->get('/user', function (Request $request) {
-//    return $request->user();
-// });
+// Articles
+use App\Http\Controllers\Articles\ArticlesController;
+
+Route::prefix('articles')->group(function () {
+    Route::post('/', [ArticlesController::class, 'store'])->name('api.articles.store');
+});
```

Comment:
Can we add some rate limiting to this route? It can be pretty strict as I don't think it's assumed to be used too frequently.

Reply:
I've updated rate limiting for `throttle:api` to be 6 requests per minute (or once every 10 seconds) which I think will more than suffice for all of the potential API endpoints.